### ISSUE TITLE
Treat Loaded Bobby Chunks as Loaded Vanilla Chunks

### ISSUE THREAD

**Issue by SmirkVG** [Johni0702/bobby#131]
Sorry for the confusing title, I'm not totally sure how to explain this. I'll do my best in layman's terms (which is all I know).

( At its core, this issue could be called an incompatibility with the pro version of the Physics Mod, but the effects apply to any mod that relies on knowing what chunks are currently loaded. )

For context, the pro version of the Physics Mod replaces the rendering of snow and water blocks with its own renderer. Essentially -- and I'm only making an educated guess -- the Physics mod "asks" Minecraft which chunks are currently rendered so that it knows where to render its physics stuff (the water and snow). 

However, the physics mod is not rendering those elements on "Bobby Chunks." For example, ocean floors will render, but not the water.
 
 
The Farsight mod does not have this problem. Physics mod elements will render on "Farsight Chunks."

In the image below, I'm using Bobby alongside the Farsight mod with my render distance set to 24, on a server with a render distance of (16? Uncertain.). Notice how the water on the left half of the screen extends into the fog but only renders within server view distance on the right half of the screen. 

As I'm sure you know, Farsight only keeps chunks in memory, your mod writes them to disk. The chunks seen in the image below were all loaded from Bobby's cache. The chunks on the left were explored with the Farsight mod installed and thus remain in memory.

![image](https://user-images.githubusercontent.com/69607038/210667982-b9895a78-9569-471d-b53b-9a5de34df85d.png)

To summarize, it appears Farsight tells the Minecraft client that its loaded cached chunks are loaded vanilla chunks, while Bobby tells the Minecraft client that its loaded cached chunks are separate. As a result, mods that rely on knowing what chunks are loaded (via asking the Minecraft client) are not aware that they can render inside "Bobby Chunks" as well.
